sonar-project.properties

```
@@ -0,0 +1,12 @@
+sonar.projectName=presenter-account-web
+sonar.projectKey=uk.gov.companieshouse:presenter-account-web
+
+sonar.host.url=${env.CODE_ANALYSIS_HOST_URL}
+sonar.login=${env.CODE_ANALYSIS_LOGIN}
+sonar.password=${env.CODE_ANALYSIS_PASSWORD}
+sonar.sources=src
+
+sonar.exclusions=**/bin/**, **/dist/**, **/coverage/**, **/node_modules/**
+sonar.tests=test
+sonar.test.inclusions=**/test/**.spec.*
+sonar.typescript.lcov.reportPaths=coverage/lcov.info
```

```suggestion
sonar.typescript.lcov.reportPaths=coverage/lcov.info

```

I think this file needs an empty line on the end due to weirdness arround hosw the pipeline injects properties.

---

This also looks like it is not reolved. 
When I view the view in the PR the trailing newline is not there.
However, when I check it out locally it is. That's strange. 
You should try adding this trailing new line and if it still doesn't work, just leave it.